toolkit/tools/imageconfigvalidator/imageconfigvalidator.go

```diff
@@ -69,6 +69,7 @@ func ValidateConfiguration(config configuration.Config) (err error) {
 
 func validatePackages(config configuration.Config) (err error) {
```

Comment:
If possible, could we get the tests for imageconfigvalidator back? Let me know if they are failing again and I can poke at it (I think the 3rd patch I did should handle the errors).

Reply:
I don't see a problem adding it back if it's not breaking.

Reply:
Ok, I tested the patch and things have shifted a bit. I'll throw in the change with the fix for the code I forgot.

Reply:
Resolved with the patch I sent you.